NitroxClient/Communication/Packets/Processors/EntityDestroyedProcessor.cs

```diff
@@ -24,27 +24,25 @@ public EntityDestroyedProcessor(Entities entities, IPacketSender packetSender)
     public override void Process(EntityDestroyed packet)
```

Comment:
Just a remark for this file as it was modified for this PR, you can remove any reference to ``packetSender`` (don't forget the "using" line also)

Reply:
It's @Measurity's changes

Reply:
👍